language/Portuguese_Brazil/strings.po

```diff
@@ -3274,3484 +3287,3523 @@ msgstr "[Nenhum]\n"
 
 #: src/emu/inpttype.ipp:665
 msgctxt "input-name"
 msgid "Dial V 4"
-msgstr ""
+msgstr "Dial V 4"
 
 #: src/emu/inpttype.ipp:666
 msgctxt "input-name"
 msgid "Dial V 5"
-msgstr ""
+msgstr "Dial V 5"
 
 #: src/emu/inpttype.ipp:667
 msgctxt "input-name"
 msgid "Dial V 6"
-msgstr ""
+msgstr "Dial V 6"
 
 #: src/emu/inpttype.ipp:668
 msgctxt "input-name"
 msgid "Dial V 7"
-msgstr ""
+msgstr "Dial V 7"
 
 #: src/emu/inpttype.ipp:669
 msgctxt "input-name"
 msgid "Dial V 8"
-msgstr ""
+msgstr "Dial V 8"
 
 #: src/emu/inpttype.ipp:670
 msgctxt "input-name"
 msgid "Dial V 9"
-msgstr ""
+msgstr "Dial V 9"
 
 #: src/emu/inpttype.ipp:671
 msgctxt "input-name"
 msgid "Dial V 10"
-msgstr ""
+msgstr "Dial V 10"
 
 #: src/emu/inpttype.ipp:676
 msgctxt "input-name"
 msgid "Track X"
-msgstr ""
+msgstr "Track X"
 
 #: src/emu/inpttype.ipp:677
 msgctxt "input-name"
 msgid "Track X 2"
-msgstr ""
+msgstr "Track X 2"
 
 #: src/emu/inpttype.ipp:678
 msgctxt "input-name"
 msgid "Track X 3"
-msgstr ""
+msgstr "Track X 3"
 
 #: src/emu/inpttype.ipp:679
 msgctxt "input-name"
 msgid "Track X 4"
-msgstr ""
+msgstr "Track X 4"
 
 #: src/emu/inpttype.ipp:680
 msgctxt "input-name"
 msgid "Track X 5"
-msgstr ""
+msgstr "Track X 5"
 
 #: src/emu/inpttype.ipp:681
 msgctxt "input-name"
 msgid "Track X 6"
-msgstr ""
+msgstr "Track X 6"
 
 #: src/emu/inpttype.ipp:682
 msgctxt "input-name"
 msgid "Track X 7"
-msgstr ""
+msgstr "Track X 7"
 
 #: src/emu/inpttype.ipp:683
 msgctxt "input-name"
 msgid "Track X 8"
-msgstr ""
+msgstr "Track X 8"
 
 #: src/emu/inpttype.ipp:684
 msgctxt "input-name"
 msgid "Track X 9"
-msgstr ""
+msgstr "Track X 9"
 
 #: src/emu/inpttype.ipp:685
 msgctxt "input-name"
 msgid "Track X 10"
-msgstr ""
+msgstr "Track X 10"
 
 #: src/emu/inpttype.ipp:690
 msgctxt "input-name"
 msgid "Track Y"
-msgstr ""
+msgstr "Track Y"
 
 #: src/emu/inpttype.ipp:691
 msgctxt "input-name"
 msgid "Track Y 2"
-msgstr ""
+msgstr "Track Y 2"
 
 #: src/emu/inpttype.ipp:692
 msgctxt "input-name"
 msgid "Track Y 3"
-msgstr ""
+msgstr "Track Y 3"
 
 #: src/emu/inpttype.ipp:693
 msgctxt "input-name"
 msgid "Track Y 4"
-msgstr ""
+msgstr "Track Y 4"
 
 #: src/emu/inpttype.ipp:694
 msgctxt "input-name"
 msgid "Track Y 5"
-msgstr ""
+msgstr "Track Y 5"
 
 #: src/emu/inpttype.ipp:695
 msgctxt "input-name"
 msgid "Track Y 6"
-msgstr ""
+msgstr "Track Y 6"
 
 #: src/emu/inpttype.ipp:696
 msgctxt "input-name"
 msgid "Track Y 7"
-msgstr ""
+msgstr "Track Y 7"
 
 #: src/emu/inpttype.ipp:697
 msgctxt "input-name"
 msgid "Track Y 8"
-msgstr ""
+msgstr "Track Y 8"
 
 #: src/emu/inpttype.ipp:698
 msgctxt "input-name"
 msgid "Track Y 9"
-msgstr ""
+msgstr "Track Y 9"
 
 #: src/emu/inpttype.ipp:699
 msgctxt "input-name"
 msgid "Track Y 10"
-msgstr ""
+msgstr "Track Y 10"
 
 #: src/emu/inpttype.ipp:704
 msgctxt "input-name"
 msgid "AD Stick X"
-msgstr ""
+msgstr "Direcional AD X"
 
 #: src/emu/inpttype.ipp:705
 msgctxt "input-name"
 msgid "AD Stick X 2"
-msgstr ""
+msgstr "Direcional AD X 2"
```

Comment:
Would it be better to just use “Alavanca Analógica X”, “Alavanca Analógica X 2”… or  “Joystick Analógica X”, “Joystick Analógica X 2”… for these?  I’d like it to be as explicit as possible, and I still think “Direcional AD” could cause unnecessary confusion.

To be honest, I don‘t think the English terminology “AD Stick” is not very clear.,  It’s just been called that in MAME for so long that people have accepted it.  A lot of English terminology in MAME would likely be changed if we were to redesign the UI now.

Reply:
Good suggestion, fixed.